### ISSUE TITLE
Meteor.rootPath and Meteor.absolutePath are the same on Windows

### ISSUE THREAD

**Issue by konard** [VeliovGroup/Meteor-root#3]
Both `Meteor.rootPath` and `Meteor.absolutePath` returns full path to 'programs/server' on Windows (meteor 1.2.1).

This is because final paths on Windows use `\\` instead of `/` for folder separator. And `split('/.meteor')` no longer finds any `'/.meteor'` in resulting path.

So it is better to use `path.sep` in `path.sep + '.meteor'` as the result argument for `split` function.

So the possible solution is to change:

`Meteor.absolutePath = __meteor_bootstrap__.serverDir.split('/.meteor')[0];` (https://github.com/VeliovGroup/Meteor-root/blob/c2978b2dd76248156aef97541905e11fa1ed1fcf/meteor-root.js#L2)

to

``` javascript
var path = Npm.require('path');
Meteor.absolutePath = __meteor_bootstrap__.serverDir.split(path.sep + '.meteor')[0];
```

or

``` javascript
var path = Npm.require('path');
Meteor.absolutePath = path.resolve('.').split(path.sep + '.meteor')[0];
```

Looks like `__meteor_bootstrap__.serverDir` is equal to `path.resolve('.')` and `process.cwd()`.

More at: http://stackoverflow.com/questions/18378809/find-absolute-base-path-of-the-project-directory-after-meteor-0-6-5/35948996#35948996
And: http://stackoverflow.com/questions/18378809/find-absolute-base-path-of-the-project-directory-after-meteor-0-6-5/35948996#comment59967680_32658007

**Comment by dr-dimitru** [comment]
Would you run tests and send PR? Otherwise I can do it myself, but later

**Comment by konard** [comment]
I updated the issue description. `path.sep` is required to solve the issue.

**Comment by konard** [comment]
Ok, I`ll try to make a PR.

**Comment by dr-dimitru** [comment]
@Konard 
Why not use Regular Expression?

``` js
Meteor.absolutePath = __meteor_bootstrap__.serverDir.split(/(\\|\/).meteor/)[0];
```

**Comment by konard** [comment]
For me it looks like the best solution is:

``` javascript
var path = Npm.require('path');
Meteor.absolutePath = path.resolve('.').split(path.sep + '.meteor')[0];
```

`__meteor_bootstrap__` requires `bootstrap` package.
`path.sep` can be extended in future with other file systems / platforms.

But `Meteor.absolutePath = __meteor_bootstrap__.serverDir.split(/(\\|\/).meteor/)[0];` can also be good enough if `bootstrap` included to the project.

**Comment by dr-dimitru** [comment]
isn't `__meteor_bootstrap__` always presented?

**Comment by konard** [comment]
Trying to run `meteor test-packages` but it does not work. https://ide.c9.io/konard/meteor-root (cloned your repository here). How do you run `Tinytest`?

**Comment by dr-dimitru** [comment]
In project's directory run:

```
meteor test-packages ./
```

**Comment by konard** [comment]
```
konard:~/workspace (master) $ meteor test-packages ./
[[[[[ Tests ]]]]]                             

=> Started proxy.                             
=> Started MongoDB.                           
=> Started your app.                          

=> App running at: http://localhost:3000/
```

This is what I get. No output in console.

**Comment by dr-dimitru** [comment]
First visit `http://localhost:3000/`

**Comment by konard** [comment]
I actually cannot do it. I have only access to ssh. And to 8080, 8081, 8082 ports from the internet.

**Comment by konard** [comment]
Ah. `meteor test-packages \. --port 8080` works. https://meteor-root-konard.c9users.io/ is live now.

**Comment by dr-dimitru** [comment]
1. Why not do it locally?
2. Forward some port, to 8080

I see you've solved it

**Comment by konard** [comment]
This code have the same results as current version (https://github.com/VeliovGroup/Meteor-root/blob/c2978b2dd76248156aef97541905e11fa1ed1fcf/meteor-root.js):

``` javascript
var path = Npm.require('path');
Meteor.rootPath     = path.resolve('.');
Meteor.absolutePath = Meteor.rootPath.split(path.sep + '.meteor')[0];
```

And the only changes required are these 3 lines. It works on windows too (tested).

**Comment by dr-dimitru** [comment]
Published as [v1.0.3](https://github.com/VeliovGroup/Meteor-root/releases/tag/1.0.3), also available via Atmosphere.

Thank you for contribution @Konard 
If you found this package useful, please do not hesitate to star it at both [GitHub](https://github.com/VeliovGroup/Meteor-root) and [Atmosphere](https://atmospherejs.com/ostrio/meteor-root). Also you may like to [Tweet](https://twitter.com/share?url=https://github.com/VeliovGroup/Meteor-root&text=Get%20MeteorJS%20root%20path) about it or share at [Facebook](https://www.facebook.com/sharer.php?u=https://github.com/VeliovGroup/Meteor-root)

Sorry for off-topic, but I saw you has knowledge in Traditional Chinese, if it's true, let me know if you're interested in translating interfaces, we're looking for help with next languages:
- French;
- German;
- Spanish;
- Chinese.

And preferred way and time to contact you.

**Comment by konard** [comment]
@dr-dimitru contact me direclty using vk or skype on this topic please.
P.S. Yes, I planned to star the project as soon as I can use it. https://github.com/linksplatform/Crawler (this project was a reason, I've started looking for a stable solution), and now meteor-root is integrated with it https://github.com/linksplatform/Crawler/commit/e2f1a81e4135c57a203c75609b2bd4b541c2d4e2.
